Contuity/Models/JotInitiativeBridge.swift

```diff
@@ -67,4 +67,25 @@ extension JotInitiative {
         /// TODO: implement read method for JotInitiative
         return nil
     }
+
+    static func getJots(initiative: String) throws -> [Jot]  {
+        var jots: [Jot] = []
```

Comment:
similar thing as above, I don't see where you are actually returning the array? Just error-ing in all cases. Maybe silently fail and remove the throws keyword or something.